pkg/csi/service/common/constants.go

```diff
@@ -81,6 +81,9 @@ const (
 	// AttributePvcNamespace represents the namespace of the PVC
 	AttributePvcNamespace = "csi.storage.k8s.io/pvc/namespace"
 
+	// AttributeStorageClassName represents name of the Storage Class.
+	AttributeStorageClassName = "csi.storage.k8s.io/sc/name"
```

Comment:
This will be supplied by the customized csi-provisioner. The csi provisioner change will be  added by @shalini-b